src/components/TopNav.js

```diff
@@ -47,39 +70,150 @@ class TopNav extends Component {
             <Redirect to="/accounts/login" />
           ) : (null)
         }
-        <Menu inverted color="black">
-          <Menu.Item as={Link} to="/">
-            <Image src={logoImage} size="mini" />
-          </Menu.Item>
-          <Menu.Item as={Link} to="/programs">
-            <FormattedMessage
-              id="app.top_nav.programs"
-              description="Button label to go to programs"
-              defaultMessage="Programs"
-            />
-          </Menu.Item>
-          <Menu.Menu position="right">
-            <Dropdown item text={userName}>
-              <Dropdown.Menu>
-                <Dropdown.Item as={Link} to="/user/settings">
+
+        <div>
+          <AppBar position="static" color="secondary">
+            <Toolbar>
+              <Grid item container direction="row" spacing={4}>
+                <Grid item xs={2}>
+                  <Box
```

Comment:
The mock called for the nav links to be centered inside areas with even widths. I did that by setting the grid item at `{2}` columns and using a `Box` inside.
The `{2}`-column width looks pretty good all the down to tablet sizes, so I kept it constant.